$MirOS: ports/net/centericq/patches/patch-src_hooks_irchook_cc,v 1.3 2007/12/23 23:27:28 bsiegert Exp $
--- src/hooks/irchook.cc.orig	Sun Dec 23 21:49:27 2007
+++ src/hooks/irchook.cc	Sun Dec 23 21:53:04 2007
@@ -1264,7 +1264,7 @@ const char * const command, const char *
 	map<string, time_t>::iterator i = irhook.pingtime.find(up(nick));
 
 	if(i != irhook.pingtime.end()) {
-	    snprintf(buf, NOTIFBUF, _("PING reply from the user: %d second(s)"), time(0)-i->second);
+	    snprintf(buf, NOTIFBUF, _("PING reply from the user: %d second(s)"), (int)(time(NULL) - i->second));
 	    buf[NOTIFBUF-1] = '\0';
 	    em.store(imnotification(imcontact(nick, irc), buf));
 	}
